### PR TITLE
Webaccess cuelist side fader

### PR DESCRIPTION
**Pull request by Itay-Lifshitz**
Hi, @mcallegari 
Cue List side fader(none, crossfade, steps) is now supported
Please review it and give me feedbacks.

### CODE REVIEW

**Comment by mcallegari** [comment]
@Itay-Lifshitz ok, almost there.
It works well except for the label colors. If you look at the original widget in the virtual console, you will notice that during playback, steps (and side fader labels) are colored in orange and blue, to indicate the current step and the next step.
Please check if this is feasible too on web interface, otherwise for now let's proceed with this.
I haven't done a code review yet. First I wanted to check if it was working as expected.
Thanks

File: ui/src/virtualconsole/vccuelist.cpp [hunk 1]
```diff
@@ -676,6 +676,8 @@ void VCCueList::slotStop()
                                             .arg(m_stopButton->palette().window().color().name()));
             m_progress->setFormat("");
             m_progress->setValue(0);
+
+            emit slotCueProgressState();
```
Comment (mcallegari) [inline ui/src/virtualconsole/vccuelist.cpp:680]: Signals are not prefixed with the "slot" keyword (slots are)
Please rename to "progressStateChanged"

File: ui/src/virtualconsole/vccuelist.cpp [hunk 2]
```diff
@@ -1116,6 +1140,58 @@ void VCCueList::slotShowCrossfadePanel(bool enable)
...
+{
+    return m_primaryTop;
+}
+
+void VCCueList::slotSideFaderButtonChecked(bool enable)
+{
+    m_crossfadeButton->setChecked(enable);
+}
+
+bool VCCueList::showSideFaderPanel()
```
Comment (mcallegari) [inline ui/src/virtualconsole/vccuelist.cpp:1182]: Please rename to "isSideFaderVisible"

File: webaccess/src/webaccess.h [hunk 3]
```diff
@@ -86,6 +86,9 @@ protected slots:
     void slotSliderValueChanged(QString val);
     void slotAudioTriggersToggled(bool toggle);
     void slotCueIndexChanged(int idx);
+    void slotCueProgressStateChanged();
+    void slotCueShowSideFaderPanel();
+    void soltCueSideFaderValueChanged();
```
Comment (mcallegari) [inline webaccess/src/webaccess.h:91]: typo "solt" -> "slot"

**Comment by mcallegari** [comment]
@Itay-Lifshitz a few things to rename (see my comments) but the rest works pretty well. Thanks

**Comment by Itay-Lifshitz** [comment]
> @Itay-Lifshitz a few things to rename (see my comments) but the rest works pretty well. Thanks

I have fixed. Please check again.

**Comment by mcallegari** [comment]
Thanks 👍

**Pull request merged**